src/styles/blocks/star.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+  width: 96px;
+  margin-left: 18px;
+}
+
+.stars__star {
+  background-image: url(../images/star-active.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+  background-image: url(../images/star.svg);
+}
```

Comment:
The background image for the inactive star should not be set on the `:last-child` pseudo-class. It should be a default background image for all the stars, and the active star image should be set based on the modifier class. Please, revise the task requirements and adjust your code accordingly.